src/ACESVGjsPanel.tsx

```diff
@@ -118,7 +118,7 @@ export class ACESVGPanel extends PureComponent<Props, PanelState> {
   initializeMappings(svgNode: SVGElement | SVGDom) {
     const svgMappings = this.props.options.svgMappings;
     let currentElements: MappedElements = { ...this.state.mappedElements };
-    currentElements = {};
+    currentElements = {}; // why does it immediately get set to {}?
```

Comment:
There is something weird going on here. `currentElements` is set to the mappedElements object, and then immediately reset to `{}`

No idea why.